engine/ingame/view.py

```diff
@@ -43,10 +40,16 @@ def __init__(
             viewport_size[1],
         )
 
-    def setup(self) -> None:
-        """Sets up the view."""
-        self.width = self.game_world.width * self.game_world.tile_width
-        self.height = self.game_world.height * self.game_world.tile_height
+    @property
+    def world_width(self) -> int:
+        return self.game_world.width * self.game_world.tile_width
+
+    @property
+    def world_height(self) -> int:
+        return self.game_world.height * self.game_world.tile_height
+
+#    def setup(self) -> None:
+#        """Sets up the view."""
```

Comment:
You know what I'm going to say here :)

Actually in this case you can't remove the function, `setup()` is required by the `View` interface. So just uncomment it and leave it as it was.